### PR TITLE
feat(calendar): expose weekdayTicks prop on TimeRange component

### PR DESCRIPTION
**Pull request by brammitch**
On the new TimeRange component, the weekday tick marks are set to Monday/Wednesday/Friday. This PR gives the user an option to change them, or remove them altogether by passing an empty array.

Updates were made to the Storybook showing both alternative options.

### CODE REVIEW

File: packages/calendar/src/types.ts [hunk 1]
```diff
@@ -209,6 +209,7 @@ export type TimeRangeSvgProps = Dimensions & { data: CalendarDatum[] } & Partial
                 square: boolean
                 role: string
                 weekdayLegendOffset: number
+                weekdayTicks: number[]
```
Comment (wyze) [inline packages/calendar/src/types.ts:212]:
```suggestion
                weekdayTicks: Array<0 | 1 | 2 | 3 | 4 | 5 | 6>
```

Let's update the type to be more strict.

File: website/src/data/components/time-range/props.js [hunk 2]
```diff
@@ -228,6 +214,24 @@ const props = [
             max: 100,
         },
     },
+    {
+        key: 'weekdayTicks',
+        help: 'define weekday tickmarks to show',
+        type: 'number[]',
```
Comment (wyze) [inline website/src/data/components/time-range/props.js:220]: Update this type here as well.

**Review by wyze — CHANGES_REQUESTED** [review]
Looks good, just what a more strict type.

Also, can you merge master or rebase. I merged in a PR containing the changes you have in `.storybook` folder.

File: website/src/data/components/time-range/props.js [hunk 3]
```diff
@@ -29,20 +29,6 @@ const props = [
...
-        type: 'string | Date',
-        required: true,
-    },
-    {
-        key: 'to',
-        group: 'Base',
-        help: 'end date',
-        type: 'string | Date',
-        required: true,
-    },
```
Comment (wyze) [inline website/src/data/components/time-range/props.js:45]: Thanks for removing my copy/paste error. 👍

Comment (brammitch) [reply website/src/data/components/time-range/props.js:45]: No problem! 👍

File: website/src/data/components/time-range/props.js (hunk 2 above)
Comment (brammitch) [reply website/src/data/components/time-range/props.js]: Made changes as suggested

File: packages/calendar/src/types.ts (hunk 1 above)
Comment (brammitch) [reply packages/calendar/src/types.ts]: Made changes as suggested

**Review by wyze — APPROVED** [review]
Thanks!

**Pull request merged**